docs/sources/linking/data-link-variables.md

```diff
@@ -49,6 +49,17 @@ Value-specific variables are available under `__value` namespace:
 
 When linking to another dashboard that uses template variables, select variable values for whoever clicks the link.
 
-`${myvar:queryparams}` - where `myvar` is a name of the template variable that matches one in the current dashboard that you want to use.
-
-If you want to add all of the current dashboard's variables to the URL, then use `__all_variables`.
+- `${myvar:queryparams}` - where `myvar` is the name of the template variable that matches a variable in the dashboard that contains the selected value you want to use.
```

Comment:
I feel like this additional clarification `"that contains the selected value"` of the variable in the dashboard only adds a bit of an unnecessary complexity to the sentence and one has to read it again to be sure what follows what. For me a question arises "Does the variable in the dashboard contain the selected value, or the dashboard itself?" if I had no idea what all of this is about and am just reading it for the first time.